### PR TITLE
Adding compatibility with MongoDB.

### PR DESCRIPTION
**Pull request by akerouanton**

### CODE REVIEW

**Comment by einenlum** [comment]
Rebase needed.

**Comment by akerouanton** [comment]
@Einenlum rebased ;)

File: src/Knp/Rad/FixturesLoad/Command/FixturesCommand.php [hunk 1]
```diff
@@ -82,16 +89,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $filters = ['*.yml'];
         }
 
-        $formaters  = $this->getFormaters($output, $input->getOption('verbose'));
+        if (is_null($type = $input->getOption('type'))) {
```
Comment (PedroTroller) [inline src/Knp/Rad/FixturesLoad/Command/FixturesCommand.php:92]: Implicit condition

**Comment by Djeg** [comment]
Rabase needed again @NiR-

**Comment by einenlum** [comment]
@NiR- Is this PR still up to date?

**Comment by akerouanton** [comment]
No idea and I didn't use mongodb and symfony together in the past 2 years ...

**Pull request closed without merging**